.github/scripts/build_info.py

```diff
@@ -75,7 +75,7 @@
     "lib/setup.py",
 ]
 # +1 to make range inclusive.
-ALL_PYTHON_VERSIONS = [f"3.{d}" for d in range(8, 11 + 1)]
+ALL_PYTHON_VERSIONS = [f"3.{d}" for d in range(8, 12 + 1)]
```

Comment:
Are we now able to drop support for 3.8? I think we'd normally be able to given our version compatibility guarantees, but there may be some external dependencies that require 3.8 that prevent us from doing so

Reply:
Not yet, [based on internal discussion](https://www.notion.so/Support-Python-3-12-2b0409def8d5475caec39f640da5fde5?d=fc42fc42fc17419081e9a52e5159126a&pvs=4) we don't tie the support of 3.12 to the deprecation of 3.8.